_api_formula_linux.json.in

```diff
@@ -0,0 +1,4 @@
+---
+layout: formula_json
```

Comment:
Because `_layouts/formula_json.json` references `site.data.formula`, you'd need to either create a new `_layouts/formula-linux_json.json` that references `site.data.formula-linux`, or add a variable to the frontmatter of both this page and `_api_formula.json.in`, then modify `_layouts/formula_json.json` to check that variable when deciding where to get formula data from.